lib/lightning/credentials/schema.ex

```diff
@@ -66,6 +66,9 @@ defmodule Lightning.Credentials.Schema do
 
       %{actual: "null", expected: ["string"]} ->
         add_error(changeset, field, "can't be blank")
+
+      _ ->
+        add_error(changeset, field, "can't be blank")
```

Comment:
I think we can make this a tiny bit clearer by changing the one above to:

```
%{actual: "null", expected: expected} when is_list(expected) ->
  add_error(changeset, field, "can't be blank")
```

I mean the amount of variance we get back from the JSON schema library is significant,  but for now I think the example I suggested covers both scenarios with "I wasn't expecting `null`"...